src/main/resources/templates/db_scripts/migrateItemCopyNumberToSingleValue.sql

```diff
@@ -1,12 +1,40 @@
 START TRANSACTION;
--- Make the array of copy numbers as a single value, take the first value from the array
-UPDATE ${myuniversity}_${mymodule}.item AS it
-  SET jsonb = jsonb_set(it.jsonb, '{copyNumber}', it.jsonb#>'{copyNumbers, 0}')
-WHERE it.jsonb->>'copyNumbers' IS NOT NULL AND jsonb_array_length(it.jsonb->'copyNumbers') > 0;
-
--- Remove the copyNumbers property even if it has zero length.
-UPDATE ${myuniversity}_${mymodule}.item AS it
-  SET jsonb = it.jsonb - 'copyNumbers'
-WHERE it.jsonb->'copyNumbers' IS NOT NULL;
+
+UPDATE pg_trigger
+  SET tgenabled = 'D'
+WHERE tgrelid IN (SELECT oid FROM pg_class WHERE relname = 'item')
```

Comment:
```suggestion
WHERE tgrelid = '${myuniversity}_${mymodule}.item'::regclass
```
There may be other tenants and modules that also have an 'item' table, always restrict to the current schema. The same for all other tgrelid occurrences.

Reply:
Thanks for catching this oversight, I was using `::regclass::oid` before and forgot to use it here (as well as the `${myuniversity}_${mymodule}`).
Updated.